package.json

```diff
@@ -38,22 +49,33 @@
     "jest": "^26.5.2",
     "jest-cli": "^26.5.2",
     "jsdoc-to-markdown": "^7.1.0",
+    "localforage": "^1.10.0",
     "metro-react-native-babel-preset": "^0.61.0",
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-native": "0.64.1",
     "react-native-performance": "^2.0.0",
-    "react-test-renderer": "16.13.1"
+    "react-test-renderer": "16.13.1",
+    "webpack": "^5.72.1",
+    "webpack-cli": "^4.9.2",
+    "webpack-merge": "^5.8.0"
   },
   "peerDependencies": {
     "@react-native-async-storage/async-storage": "^1.15.5",
+    "expensify-common": ">=1",
     "localforage": "^1.10.0",
-    "react": "^17.0.2",
+    "react": ">=17.0.2",
     "react-native-performance": "^2.0.0"
   },
   "peerDependenciesMeta": {
     "react-native-performance": {
       "optional": true
+    },
+    "@react-native-async-storage/async-storage": {
+      "optional": true
+    },
+    "localforage": {
+      "optional": true
```

Comment:
`async-storage` and `localforage` are optional to let consumers use one or the other